frame/contracts/src/wasm/mod.rs

```diff
@@ -182,6 +182,12 @@ mod tests {
 		gas_left: u64,
 	}
 
+	#[derive(Debug, PartialEq, Eq)]
```

Comment:
I think this module has grown so that it deserves its own file.

Reply:
Not which module you are talking about. You mean the wasm unit tests which are located in the wasm root module?